src/hooks/useWalletCloudBackup.js

```diff
@@ -0,0 +1,77 @@
+import { useNavigation } from '@react-navigation/core';
+import { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import walletBackupTypes from '../helpers/walletBackupTypes';
+import walletLoadingStates from '../helpers/walletLoadingStates';
+import { saveBackupPassword } from '../model/keychain';
+import { addWalletToCloudBackup, backupWalletToCloud } from '../model/wallet';
+import { setIsWalletLoading, setWalletBackedUp } from '../redux/wallets';
+import useWallets from './useWallets';
+import logger from 'logger';
+
+export default function useWalletCloudBackup({
+  wallet_id,
```

Comment:
can this be camelCase (will need to update in BackupIcloudStep line 259 and BackupConfirmPasswordStep line 186)